i18n/language/language.go

```diff
@@ -17,23 +19,35 @@ type Language struct {
 	*PluralSpec
 }
 
+// NewLanguage returns new Language with specified tag and spec.
+func NewLanguage(tag string, spec *PluralSpec) *Language {
```

Comment:
I don't see the benefit of having a factory method at this point. The factory method doesn't do anything special so I would rather not add it unless it becomes necessary.